mail/src/test/java/org/jboss/as/mail/extension/MailTransformersTestCase.java

```diff
@@ -58,6 +62,13 @@ public MailTransformersTestCase() {
         super(MailExtension.SUBSYSTEM_NAME, new MailExtension());
     }
 
+    @BeforeClass
+    public static void assumeJavaxNamespace() {
+        //noinspection ConstantConditions
+        Assume.assumeTrue("Ignoring test that requires javax.* namespace",
```

Comment:
It should be ignoring a test that requires jakarta.* namespace

Reply:
No, the test requires that the javax namespace be in effect.  It fails in the wildfly-mail-jakarta module because it uses jakarta.* but the legacy controller needs javax.

Reply:
Well, my last response didn't address your valid point. I'll make it "Ignoring test when run using the jakarta.* namespace EJB API"